### ISSUE TITLE
Install failing on GCP

### ISSUE THREAD

**Issue by grugnog** [openedx-unsupported/configuration#4770]
When doing an install of open-release/hawthorn.1 on a clean Google Cloud Ubuntu 16.06 using the native.sh install script I get the following error:

```
Traceback (most recent call last):                                                                                                              
  File \"./manage.py\", line 118, in <module>                                                                                                   
    startup.run()                                                                                                                               
  File \"/edx/app/edxapp/edx-platform/lms/startup.py\", line 19, in run                                                                         
    django.setup()                                                                                                                              
  File \"/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/__init__.py\", line 27, in setup                                 
    apps.populate(settings.INSTALLED_APPS)                                                                                                      
  File \"/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/apps/registry.py\", line 108, in populate                        
    app_config.import_models()                                                                                                                  
  File \"/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/apps/config.py\", line 202, in import_models                     
    self.models_module = import_module(models_module_name)                                                                                      
  File \"/usr/lib/python2.7/importlib/__init__.py\", line 37, in import_module                                                                  
    __import__(name)                                                                                                                            
  File \"/edx/app/edxapp/edx-platform/common/djangoapps/student/models.py\", line 54, in <module>                                               
    from lms.djangoapps.certificates.models import GeneratedCertificate                                                                         
  File \"/edx/app/edxapp/edx-platform/lms/djangoapps/certificates/models.py\", line 69, in <module>                                             
    from lms.djangoapps.instructor_task.models import InstructorTask                                                                            
  File \"/edx/app/edxapp/edx-platform/lms/djangoapps/instructor_task/models.py\", line 23, in <module>                                          
    from boto.exception import BotoServerError                                                                                                  
  File \"/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/boto/__init__.py\", line 1216, in <module>                              
    boto.plugin.load_plugins(config)                                                                                                            
AttributeError: 'module' object has no attribute 'plugin'         
```

This appears to have the same cause as #3766, which is that GCP installs a global /etc/boto.cfg file that interferes with the venv'ed code. This was addressed by #3792 however it appears that this fix was incomplete.

I have confirmed that the change from #3792 is present in the Ansible code being run by the native.sh script.
I have also confirmed that deleting /etc/boto.cfg allows the playbook to finish successfully.

**Comment by nedbat** [comment]
@grugnog Can you make a pull request with the fix?  We don't test on GCP.

**Comment by grugnog** [comment]
@nedbat I can try, although I am not very familiar with the EdX codebase, so it's hard to see why the previous fix is not working in this case. Any ideas?

**Comment by nedbat** [comment]
@grugnog I suggest asking on the mailing lists or Slack for people who can help: https://open.edx.org/getting-help

**Comment by OmarIthawi** [comment]
@grugnog This PR fixes it: #4812 

cc: @nedbat
